src/main/java/rx/Single.java

```diff
@@ -2410,10 +2412,12 @@ public void onNext(T t) {
         Observer<T> observer = new Observer<T>() {
             @Override
             public void onCompleted() {
+                // deliberately ignored
             }
 
             @Override
             public void onError(Throwable e) {
```

Comment:
Not sure if this is covered in PMD, but usually like to also name the param `ignored`